.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,49 @@
+---
+name: 'Libp2p Release'
+about: 'Start a new libp2p release.'
+---
+
+## 🗺 What's left for release
+
+<List of items with remaining PRs and/or Issues to be considered for this release>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by scripts/mkreleaselog >
+
+## ✅ Release Checklist
+
+- [ ] **Stage 0 - Automated Testing**
+    - [ ] Fork a new `release-vX.Y.Z` branch from `master`, freezing master.
+    - [ ] Make sure local tests are passing.
+- [ ] **Stage 1 - Upstream Testing**
```

Comment:
Can we guarantee libp2p interop tests work? That basically includes:
- update `go-libp2p-daemon`
- generate binary for https://github.com/libp2p/npm-go-libp2p-dep
- update interop [dep](https://github.com/libp2p/interop/blob/master/package.json#L42)

I would like to have all these automated, but I think the way to go will be to use the testground for interop in the future

Reply:
Ah, yes. SGTM.

Reply:
Done.